docs/status.md

```diff
@@ -119,7 +119,7 @@ one of the following tracking labels.
 | recv                     | yes           | revisit      | infeasible     | no              | no          |
 | reduce                   | yes           | revisit      | yes            | revisit         | no          |
 | reduce_precision         | no            | yes*         | yes*           | yes             | no          |
-| reduce_scatter           | no            | revisit      | no             | no              | no          |
+| reduce_scatter           | yes           | revisit      | no             | no              | no          |
```

Comment:
Given the the found in description, will we set the "Type Inference" as "infeasible" here?